.gitignore

```diff
@@ -54,3 +54,7 @@ packages/**/*.map
 
 # temporary test files
 test-assets/
+
+# ignore scaffolded files due to tests of init
+test/init/*
```

Comment:
Can get rid of this.

Reply:
No, but we need to add path to tests' new location.

Reply:
Yes need to be moved there.